### ISSUE TITLE
No specification between AND and OR in the prerequisite field of popups

### ISSUE THREAD

**Issue by fgormley1** [Ryu-From-Streets/classcompass#42]
When a course has multiple prerequisites the popups do not specify whether both courses are required or if they are interchangeable. In the course catalog it may say the prerequisites are course1 AND course2 or it might say the prerequisites are course1 OR course2 which are very different and specifying this could help the user navigate their courses more easily. 

Example: in the course catalog COMPSCI 335 lists its prerequisites are 220 or 230 with a grade of C or better, this is listed in your popups as Prerequisites: COMPSCI220 COMPSCI230

in the course catalog COMPSCI 515 lists its' prerequisites as: Prerequisite: COMPSI 240 and 250 with a grade of C or better in both. Your popups list it as COMPSCI240 COMPSCI250

TLDR : There is no distinction btwn and and or in the popup prerequisites.

**Comment by Ryu-From-Streets** [comment]
Resolved through new array structure. "Option" and "Any Of" nodes indicate OR, and otherwise all nodes connected to a node are AND.
